tutorials/plugins/editor/inspector_plugins.rst

```diff
@@ -8,17 +8,48 @@ editing properties. This tutorial explains how to use the
 :ref:`class_EditorInspectorPlugin` and :ref:`class_EditorProperty` classes to
 write such plugins with the example of creating a custom value editor.
 
-.. note::
+Setup
+-----
+
+Just like :ref:`doc_making_plugins`, we start out by making a new plugin,
+getting a ``plugin.cfg`` file created, and start with our
+:ref:`class_EditorPlugin`.  However, instead of using
+``add_custom_node`` or ``add_control_to_dock`` we'll use
+``add_inspector_plugin``.
+
+
+.. tabs::
+  .. code-tab:: gdscript GDScript
+    # MyEditorPlugin.gd
+
+    tool extends EditorPlugin
+
+    var plugin: EditorInspectorPlugin
+
+    func _enter_tree() -> void:
```

Comment:
Examples shouldn't use static typing as GDScript is primary dynamic language. It helps to avoid confusion for the doc users. Static is only optional way of using GDScript

Reply:
Makes sense, I'll get this updated!